datasets/sentinel-products-ca-mirror.yaml

```diff
@@ -0,0 +1,30 @@
+Name: Sentinel Products Canada Mirror
```

Comment:
@willmacs do you want to consider a more specific name like Near Real Time Sentinel Data for Canada etc.?

Reply:
The thing is, not all sensors offer NRT. I am open to further suggestions.

Reply:
ok changed to "Sentinel Near Real-time Canada Mirror"